vignettes/epiparameter.Rmd

```diff
@@ -0,0 +1,212 @@
+
+## Adding library entries
+
+If a set of epidemiological parameter has been inferred and known to the user but has not yet been incorporated into the {epiparameter} database, these parameters can be manually added to the library.
+
+To add entries to the library the `bind_epiparam()` function appends a row to an existing `<epiparam>` object using either an `<epiparam>`, `<epidist>` or data frame.
+
+```{r add-to-library}
+bind_epiparam(epiparam = epi_dist_db, epi_obj = ebola_incubation)
+bind_epiparam(epiparam = epi_dist_db, epi_obj = as_epiparam(ebola_incubation))
+```
+
+Note that this only adds the parameters to the library (`<epiparam>` object) in the environment, and does not save to the database file in the package.
+
+## Distribution functions
+
+`<epidist>` objects store distributions, and mathematical functions of these distribution can easily be extracted directly from them. It is commonly required to extract the probability density function, cumulative distribution function, quantile or generate random numbers from the distribution in the `<epidist>` object. The distribution functions in {epiparameter} allow users to easily access these aspects.
+
+```{r epidist-dist-methods}
+density(ebola_incubation, at = 0.5)
+cdf(ebola_incubation, q = 0.5)
+quantile(ebola_incubation, p = 0.5)
+generate(ebola_incubation, times = 10)
+```
+
+## Plotting epidemiological distributions
+
+`<epidist>` objects can easily be plotted to see the PDF and CDF of distribution.
+
+```{r plot-epidist, fig.width=7}
+plot(ebola_incubation)
+```
+
+The default plotting range for time since infection is from zero to ten days. This can be altered by specifying the `day_range` argument when plotting an `<epidist>` object.
+
+```{r plot-epidist-dayrange, fig.width=7}
+plot(ebola_incubation, day_range = 1:25)
+```
+
+This plotting function can be useful for visually comparing epidemiological distributions from different publications on the same disease. In addition, plotting the distribution after manually creating an `<epidist>` help to check that the parameters are sensible and produce the expected distribution. 
+
+## Parameter conversion and extraction
+
+### Conversion
+
+Parameters are often reported as mean and standard deviaton (or variance). These can be (analytically) converted to the parameters of the distribution using the conversion functions in the package. These have the naming convention `summary_stats2params()`. For example `gamma_meansd2shapescale()` or `lnorm_meansd2musigma()`.
+
+We also provide conversion functions in the opposite direction, parameters to summary statistics. These follow the same naming convention, for example `weibull_shapescale2meansd()`.
+
+### Extraction
+
+The functions `extract_param()` handles all the extraction of parameter estimates from summary statistics. The two extractions currently supported in {epiparameter} are from percentiles and from median and range.
+
+## Contributing to {epiparameter}
```

Comment:
I think it would be best to get this initial release out without linking to the WHO Collaboratory, and then subsequent releases can be linked once we have adopted their database.